### DISCUSSION TITLE
bandmap

### DISCUSSION THREAD

**Discussion by TecnoTalarn** [foldynl/QLog#474] [Ideas]
Hello, I thought that the frequency could be made clickable in the Bandmap, and thus be able to quickly browse the band with the mouse.
including, taking advantage of the wheel, if we are on the bandmap that it makes us move the dial...

![bandmap](https://github.com/user-attachments/assets/cee6c8f6-612d-4f5d-8c6c-296b500d5876)

**Comment by foldynl** [comment]
The question is, what benefit does having a clickable frequency bring? As it is designed now, the bandmap is an overview of callsigns, and I click on these callsigns. The bandmap shouldn't be used for tuning frequencies in some step.

Could you please specify more regarding the wheel?

**Comment by TecnoTalarn** [comment]
I don't know to what extent it is necessary, or useful, to have some way to go to a frequency from the computer screen.
I'm quite used to using WFview to operate the Icoms, both remotely and if I have them on the shack table.
Now, with the Yaesu, I don't have the tools to work like that. The integration is not as good as Icom, and only the FLrig allows me to move the dial from the screen, but it is also not as operational as I would like.
If QLOG had a dial, either in the bandmap, or maybe in the RIG window, I'd have everything I'm using on my desktop, regardless of what device I'm using.

The wheel, I mean that the dial can be used with the mouse wheel, but it may not be a very good idea, since it is also used to move up and down the lists and the bandmap.
